prepare-build.sh

```diff
@@ -47,10 +47,16 @@ function pull_dir {
         [ "$1" != "." ]   && pushd $1 > /dev/null
         git symbolic-ref HEAD &> /dev/null
         if [ $? -eq 0 ] ; then
-            echo -e "\e[32mPulling $1\e[39m"
-            git pull --rebase
+            if git remote get-url upstream > /dev/null
```

Comment:
Could you add a comment here explaining pretty much what you explained in the PR description ? To act as minimal documentation for this.

Reply:
Done!